presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -215,12 +227,12 @@ private VerifierQueryEvent buildEvent(Validator validator)
                 queryPair.getTest().getCatalog(),
                 queryPair.getTest().getSchema(),
                 queryPair.getTest().getQuery(),
-                optionalDurationToSeconds(test.getCpuTime()),
+                checkCpu ? optionalDurationToSeconds(test.getCpuTime()) : null,
```

Comment:
Instead of doing this here, it should be done in the `HumanReadableEventClient`. We may still want to log those values. Just not print a warning about the ratio